sql/moz-fx-data-shared-prod/stub_attribution_service_derived/dl_ga_triplets_v1/query.sql

```diff
@@ -0,0 +1,35 @@
+WITH historical_triplets AS (
+  SELECT
+    dl_token,
+    ga_client_id,
+    stub_session_id,
+    first_seen_date,
+  FROM
+    stub_attribution_service_derived.dl_ga_triplets_v1
+),
+new_downloads AS (
+  SELECT DISTINCT
+    mozfun.ga.nullify_string(jsonPayload.fields.dltoken) AS dl_token,
+    mozfun.ga.nullify_string(jsonPayload.fields.visit_id) AS ga_client_id,
+    mozfun.ga.nullify_string(jsonPayload.fields.session_id) AS stub_session_id,
+    @download_date AS first_seen_date,
+  FROM
+    `moz-fx-stubattribut-prod-32a5`.stubattribution_prod.stdout
+  WHERE
+    DATE(timestamp) = @download_date
```

Comment:
nit: these logs get truncated right? Might be worth adding a clause similar to https://github.com/mozilla/bigquery-etl/pull/3352 to prevent dangerous backfills?
```suggestion
  FROM
    `moz-fx-stubattribut-prod-32a5`.stubattribution_prod.stdout
  WHERE
    DATE(timestamp) = @download_date
    AND IF(
        DATE_DIFF(CURRENT_DATE(), @download_date, DAY) > 30,
        ERROR("Data older than 30 days has been removed"),
        NULL
  ) IS NULL
```

Reply:
Funny, I did have that at first! But since this does a `FULL OUTER JOIN` with itself, that will have no negative effects, so I removed it to simplify the query. Happy to include it again if you think that will clarify the query (or if there's a failure mode I'm missing).